openpype/settings/defaults/project_settings/deadline.json

```diff
@@ -19,6 +19,21 @@
                 "deadline"
             ]
         },
+        "HoudiniSubmitDeadline": {
+            "enabled": true,
+            "active": true,
+            "use_published": true,
+            "priority": 50,
+            "render_chunk_size": 1,
+            "group": "none",
+            "export_priority": 50,
+            "export_chunk_size": 10,
+            "export_group": "none",
+            "department": "",
+            "env_allowed_keys": [],
+            "env_search_replace_values": {},
```

Comment:
I feel like this is a reasonable feature - maybe not entirely 'required' for this particular PR and potentially applicable to other DCCs as well.

1. Could you give an example of what type of search/replace you'd configure?
2. Would it make sense taking this out of this PR and potentially into a separate PR that implements this particular behavior on a higher level, e.g. on `AbstractSubmitDeadline` so other implementations would match + benefit from it?

Reply:
yeah I will split it, I basically wanted to bring some of the features that already exist in other Deadline submitters to Houdini as well. I'm really not making use of the env patch features but it would certainly be helpful for remaps of paths or other mapping differences from local / farm environment